devito/symbolics/extended_sympy.py

```diff
@@ -796,6 +794,51 @@ def __str__(self):
     __repr__ = __str__
 
 
+class VectorAccess(Expr, Pickable):
+
+    """
+    Represent a vector access operation at high-level.
+    """
+
+    def __new__(cls, *args, **kwargs):
+        return Expr.__new__(cls, *args)
+
+    def __str__(self):
+        return f"VL4<{self.base}>"
```

Comment:
Is this always "4"? If so, why?

Reply:
Yeah, what is "VL4"?

Reply:
VL4 is basically only because I use it to represent float4, but yes, I agree it's nonsensical here, so I'll drop it